<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHost</class>
 <widget class="QDialog" name="frmHost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主机管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/main.png</normaloff>:/image/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;宋体&quot;;</string>
  </property>
  <widget class="QListWidget" name="listHost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>421</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labHostConnectValue2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>撤防命令:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>主机编号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txtHostType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>删除(&amp;D)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labHostType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>主机类型:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHostConnectValue2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHostConnectValue1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labHostConnectValue1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>布防命令:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>修改(&amp;U)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/excel.png</normaloff>:/image/excel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHostID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>添加(&amp;A)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labHostName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>主机名称:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHostName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listHost</tabstop>
  <tabstop>txtHostType</tabstop>
  <tabstop>txtHostID</tabstop>
  <tabstop>txtHostName</tabstop>
  <tabstop>txtHostConnectValue1</tabstop>
  <tabstop>txtHostConnectValue2</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnUpdate</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
